x-pack/plugins/lens/public/app_plugin/save_modal_container.tsx

```diff
@@ -64,41 +62,7 @@ export function SaveModalContainer({
     savedObjectId = lastKnownDoc.savedObjectId;
   }
 
-  const {
```

Comment:
this code was unnecessary - basically instead of using the lastKnownDoc prop from the app, we were creating it again here.